modin/pandas/series_utils.py

```diff
@@ -485,115 +516,122 @@ def _default_to_pandas(self, op, *args, **kwargs):
 class DatetimeProperties(ClassLogger):
     def __init__(self, series):
         self._series = series
+        self._series_type = type(series)
         self._query_compiler = series._query_compiler
 
     @property
     def date(self):
-        return Series(query_compiler=self._query_compiler.dt_date())
+        return self._series_type(query_compiler=self._query_compiler.dt_date())
```

Comment:
maybe like this?
```suggestion
        return self._series.__constructor__(query_compiler=self._query_compiler.dt_date())
```
it seems to be the pattern we're using everywhere in the probject